kge/model/complex.py

```diff
@@ -60,3 +61,6 @@ def __init__(
             configuration_key=configuration_key,
             init_for_load_only=init_for_load_only,
         )
+    
+    def penalty(self, **kwargs) -> List[Tensor]:
+        return super().penalty(is_complex= True, **kwargs)
```

Comment:
ComplEx isn't the only model that uses complex embeddings. Also, I suggest to move the is_complex flag to lookup embedder instead.

Reply:
Hi Prof Gemulla, 

Instead of using is_complex, I created a new configuration in the YAML file, which is vector_space (accepts real and complex values). The computing N3 will base on the value of vector_space configuration to compute the penalty for embedding vectors.

Please let me know what you think about this change. 

For the other model - RotatE, I will change it when we finalize the final approach.